### ISSUE TITLE
the HTML title for the TOPICS BETA category should be 'gitterHQ – Topics beta'

### ISSUE THREAD

**Issue by grahamperrin** [gitterHQ/gitter#1650]
1. https://gitter.im/gitterHQ/topics/categories/topics-beta

## Expected

- _gitterHQ – Topics beta_

­– in the title bar. With an en dash. 

## Actual result

- _Gitter_

– lack of distinction, which is troublesome when working with multiple tabs. 

In the example below, only after clicking Gitter does it become clear that it's not the Gitter home page: 

![title without distinction](https://cloud.githubusercontent.com/assets/192271/24580047/190bb366-16f9-11e7-8aae-894676666d1f.png)

## Thoughts

I wondered whether the title should be more verbose, for example: 

_gitterHQ – Topics – Topics beta_

– but the 'Topics –' in the middle is probably excessively meta, superfluous. 

(A good topic will be suitably titled by its author, and so on.)

Topics, and categorisations therein, are relatively new to me, so I'm open to suggestions.

**Comment by MadLittleMods** [comment]
Closing in favor of https://github.com/gitterHQ/gitter/issues/1651
